### DISCUSSION TITLE
建议添加CNKI翻译服务源

### DISCUSSION THREAD

**Discussion by TangBoheng** [immersive-translate/immersive-translate#1973] [Ideas]
### 新功能 | New Feature

建议添加CNKI翻译服务。

### 为什么您想要添加这些功能 | Why you want to add these features

使用科技类翻译源词语翻译准确度有问题。并且语句不通顺。尤其时PDF论文时很影响阅读体验。使用Zotero翻译插件时翻译源默认CNKI翻译服务，免费准确很好用，只是限定长度。希望能添加CNKI的源。

**Comment by littledgg** [comment]
也想提issue的，发现有人提了，过来支持一下，CNKI翻译是免费不限额的，算是印象中知网唯一做的好事了🥱
https://fanyi.cnki.net/serviceIntroduce/index.html
这是CNKI的文本翻译API文档，对着固定的统一的API网址发送HTTP请求就行，非常简单的API接口，希望能加入进来。

**Reply by theowenyoung** [reply]
该网址目前 404

**Reply by littledgg** [reply]
奇怪，我还是能正常访问，我给网页截了个图
文本翻译API地址
http://192.168.52.38:5102/translator/translate（待定）
![image](https://github.com/user-attachments/assets/5f99b77c-06a8-4a72-bbd5-1f28038243a5)
